conduct.md

```diff
@@ -4,66 +4,85 @@ layout: page
 
 ## Code of Conduct
 
-DjangoCon Europe aims to be a welcoming event where people meet in a friendly environment.
-Accordingly, all participants are expected to show respect and courtesy to other participants
-throughout the conference.
+All delegates/attendees, speakers, exhibitors, organizers and
```

Comment:
"Everybody who participates in DjangoCon Europe in one way or another, including …" might make this even more clear, and remove any roles we might forget now or that will be introduced later on.

Reply:
👍